src/API/GitLab/GitLabAPI.php

```diff
@@ -16,6 +17,8 @@ class GitLabAPI extends WebAPI
 {
     const SERVICE_NAME = 'gitlab';
     const GITLAB_TOKEN = 'GITLAB_TOKEN';
+    const GITLAB_CONFIG_PATH = 'command.build.provider.git.gitlab_url';
```

Comment:
Using config for this is a good idea. The `command.build` namespace should be reserved for configuration that sets actual build tools plugin command options.  `command.build.project.create.shortcuts` also gets this wrong.  Maybe `build-tools.provider.git.gitlab.url` would be a good choice.

Reply:
Ok. Updated per your suggestion (and what you say makes sense).